src/components/views/AppSettingsView.tsx

```diff
@@ -810,4 +843,17 @@ class AppSettingsView extends React.Component<Props, State> {
 
 }
 
-export default withStyles(styles)(AppSettingsView);
\ No newline at end of file
+/**
+ * Maps redux state to props
+ *
+ * @param state redux state
+ */
+const mapStateToProps = (state: ReduxState) => ({
+  selectedContentVersion: state.contentVersion.selectedContentVersion
```

Comment:
AppSettingView is only used in ApplicationEditor which is already Redux-aware and contains selectedContentVersion, move the selectedContentVersion to props and pass it to the view from there instead of injecting it directly here